yarn-project/noir-private-kernel/src/types/private_kernel_init_types.ts

```diff
@@ -2,7 +2,7 @@
 
 /* eslint-disable */
 
-export type FixedLengthArray<T, L extends number> = L extends 0 ? never[]: T[] & { length: L }
+export type FixedLengthArray<T, L extends number> = L extends 0 ? never[] : T[] & { length: L };
```

Comment:
Is this from prettier or why extra spaces?

Reply:
yes, I just run yarn format on the repo. It indeed looks weird 🤔